### ISSUE TITLE
"For Next Release" Label + Pull Request Checklist

### ISSUE THREAD

**Issue by NAnguiano** [dondi/GRNsight#440]
This is more of a proposition than an issue.

I think it may be nice to have some sort of label that indicates that a particular issue is aimed to be completed before the next master release. This way, we can track issues that need to be done before a release is completed. This can be used to tag any existing bugs major enough that we'd want to fix them before a release (such as #417 and #364), or any features that we really want to have in the next release. 

I guess that's pretty much the same thing as priority 0, but it could be used just in the weeks leading up to a release? I don't know, just let me know what you think.

The second thing I want to propose is a "Pull Request Checklist". Just a small list of things that a pull request must satisfy before it is accepted into beta. It may not be an 100% complete rigorous test, but just something to ensure that at least the basics are functioning as expected. I think it would make checking pull requests for accuracy a bit faster and easier (because everything's easier with lists :smile:), and maybe catch some early bugs. 

Of course, you'd want to check that whatever functionality the pull request added is working as expected (and that the pull request actually contains the functionality it's supposed to contain), but this would be good to check after you've confirmed that the pull request's functionality is working fine and before the pull request is accepted.

The wording would change as functionality was added, but this is what I'm thinking of for now:

Pull Request Checklist
- [ ] All Tests Pass
- [ ] All Loaded Demos look as Expected
- [ ] Reload works as expected
- [ ] Loading File with Error brings up error modal, and it looks as expected
- [ ] Loading File with Warning brings up warnings modal
- [ ] Viewport Option Changing works as expected
- [ ] Viewport Size Changing works as expected
- [ ] Hide/Show Weights works as expected
- [ ] Force Sliders change the number above them and have an effect on the graph
- [ ] Locking/Unlocking/Resetting/Undoing the force graph parameters works as expected
- [ ] Weighted graph loaded with the "default to black edges" option checked appears as expected
- [ ] Import works for both SIF and GraphML
- [ ] Graph can be exported to SIF and GraphML

If something fails, you can point out exactly where it fails. I'm going to run through this and see how long it takes, but I'm thinking it's thorough enough to catch any major bugs without being so labor-intensive that no one would want to do it. I'll edit this with how long this takes me once I review @eileenchoe pull request.

Edit: formatting is hard

**Comment by kdahlquist** [comment]
We agreed at the meeting that the pull request checklist is working and that we would create a new issue tag called "for next release" so that we could definitively say which solved issues are going into the next release.

**Comment by NAnguiano** [comment]
The pull request checklist is now put on [this wiki page](https://github.com/dondi/GRNsight/wiki/Pull-Request-Checklist).
